Samples.Objc/Samples.ObjC/API/Endpoints/FetchAccessTokenEndpoint.m

```diff
@@ -1,37 +1,32 @@
 //
 //  FetchAccessTokenEndpoint.m
-//  PayPalCheckout-Samples-iOS-Objc
+//  PayPalNativeCheckoutObjC
 //
-//  Created by Haider Khan on 5/18/20.
-//  Copyright © 2020 PayPal. All rights reserved.
+//  Created by Nguyen, The Nhat Minh on 3/14/21.
```

Comment:
Please revert this, or add the copyright header `Copyright © 2020 PayPal. All rights reserved.`